yarn-project/txe/src/util/txe_public_contract_data_source.ts

```diff
@@ -13,6 +13,7 @@ import {
 import type { TXE } from '../oracle/txe_oracle.js';
 
 export class TXEPublicContractDataSource implements ContractDataSource {
```

Comment:
Please add here a comment on why do we have this modified contract data source.

Reply:
This thing might go away, don't really know why it's done like this